internal/cli/util/version.go

```diff
@@ -114,3 +115,14 @@ func GetKubeBlocksDeploy(client kubernetes.Interface) (*appsv1.Deployment, error
 	}
 	return &deploys.Items[0], nil
 }
+
+// GetDockerVersion get Docker Version
+func GetDockerVersion() string {
+	// exec cmd to get output from docker info --format '{{.ServerVersion}}'
+	cmd := exec.Command("docker", "info", "--format", "{{.ServerVersion}}")
+	out, err := cmd.Output()
+	if err != nil {
```

Comment:
If docker is not running, this command will throw error:
```
Cannot connect to the Docker daemon at unix:///Users/ldm/.docker/run/docker.sock. Is the docker daemon running?
```

This error is useful, kbcli can throw it directly.